lms/djangoapps/teams/views.py

```diff
@@ -877,6 +876,14 @@ def post(self, request):
         except User.DoesNotExist:
             return Response(status=status.HTTP_404_NOT_FOUND)
 
+        course_module = modulestore().get_course(team.course_id)
+        if 'max_team_size' in course_module.teams_configuration \
```

Comment:
As a style note, we generally try to avoid the use of "\". I don't feel strongly about it, but it is one of our conventions that you should be aware of.